validation-test/BuildSystem/extractsymbols-default-behaviour.test

```diff
@@ -0,0 +1,76 @@
+# REQUIRES: standalone_build
+# REQUIRES: OS=macosx
+
+# RUN: %empty-directory(%t)
+# RUN: mkdir -p %t
+# RUN: split-file %s %t
+
+# RUN: mkdir -p %t/destdir
```

Comment:
Can you add a comment here explaining that:

1. Even though this is a test using dry-run, the symbol extraction code is not just echoed.
2. As a result, we want to make sure that the current behavior is at least tested and can take advantage of the (1) to generate a "simulated" build root and make sure that we do what we expect.

Reply:
Added comment in b903a6cc658511ea333b357f05cbfd6362f17a17